pom.xml

```diff
@@ -165,25 +165,32 @@
           <plugin>
             <groupId>org.apache.maven.plugins</groupId>
             <artifactId>maven-javadoc-plugin</artifactId>
-            <version>2.9.1</version>
-	    <configuration>
+            <version>3.0.0</version>
+            <configuration>
               <additionalparam>-Xdoclint:none</additionalparam>
-	    </configuration>
-	    <executions>
+            </configuration>
+            <executions>
               <execution>
                 <id>attach-javadocs</id>
                 <goals>
                   <goal>jar</goal>
                 </goals>
               </execution>
+              <execution>
+                <id>aggregate</id>
+                <goals>
+                  <goal>aggregate</goal>
```

Comment:
What's this for? `javadoc:aggregate` is designed for aggregating docs from multi-module projects, but openprops only has one module. See https://maven.apache.org/plugins/maven-javadoc-plugin/examples/aggregate.html